Scripts/Python/ki/xKIChat.py

```diff
@@ -1208,6 +1217,63 @@ def ChangePassword(self, newPassword):
             return
         PtChangePassword(newPassword)
 
+    ## Overrides the chat area text colors to be a single user-selected color
+    def SetTextColor(self, arg1, arg2=None, arg3=None):
+
+        clamp = lambda val, minVal, maxVal: max(minVal, min(val, maxVal))
+
+        if not arg1:
+            self.chatMgr.AddChatLine(None, PtGetLocalizedString("KI.Errors.MalformedChatSetTextColorCmd"), kChat.SystemMessage)
+            return
+
+        try:
+            colorRed = float(arg1)
+            colorBlue = float(arg2)
+            colorGreen = float(arg3)
+        except ValueError:
+            pass # arguments weren't valid float numbers, so we will try treating arg1 as a string below
```

Comment:
PEP8 says comments shall be on their own line.

Reply:
Fair, I'll modify. I haven't had any insomnia lately that would drive me to read the PEP8 documentation. 😛